operatorconfig/driverconfig/powerflex/v2.2.0/controller.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: v1
```

Comment:
Why are we adding support for powerflex driver 2.2? Current operator supports n-2 versions of driver.. If we are following the same, we should not have folder related to 2.2

Reply:
removed it